tests/MenderAPI/deployments.py

```diff
@@ -44,6 +44,9 @@ def get_deployments_base_path(self):
     def upload_image(self, filename, description="abc"):
         image_path_url = self.get_deployments_base_path() + "artifacts"
 
+        logger.info("Sleep 60 seconds to be sure minio is up and running...")
+        time.sleep(60)
```

Comment:
In the tests that use the client (most of them) we will be already waiting for a minute to get SSH before arriving here.

I suggest we move this sleep to the fixtures in `common_setup`, adding it only to the fixtures that do not wait for the client (i.e. the ones who do not call to the esoteric function `ssh_is_opened()`)

Reply:
Also, there are tests which will call upload_image several times, making the workaround here even more aggressive,

Reply:
I reduced the sleeping to 15 seconds (which are enough); in my tests moving it to the common_setup did not work as expected and still triggered the issue.

Reply:
That is very strange. But if you have tried it I will not argue. Let's merge this workaround as-is.

I still hope to be able to have a proper fix soon and revert it